client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/checklist-item.tsx

```diff
@@ -58,7 +58,10 @@ const ChecklistItem = ( { task, isPrimaryAction }: { task: Task; isPrimaryAction
 							/>
 						</div>
 					) }
-					<span className="launchpad__checklist-item-text">{ title }</span>
+					<div className="launchpad__checklist-item-text-container">
+						<span className="launchpad__checklist-item-text">{ title }</span>
+						{ subtitle ? <p className="launchpad__checklist-item-subtext">{ subtitle }</p> : null }
```

Comment:
Nitpick: This could be simplified a little bit.
```suggestion
						{ subtitle && <p className="launchpad__checklist-item-subtext">{ subtitle }</p> }
```